lib/wallet/api/http/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -716,6 +719,16 @@ import qualified Network.Wai.Handler.Warp as Warp
 import qualified Network.Wai.Handler.WarpTLS as Warp
 
 
+
+-- | Allow configuring which port the wallet server listen to in an integration
+-- setup. Crashes if the variable is not a number.
+walletListenFromEnv :: Show e
+    => (String -> IO (Maybe (Either e Port))) ->  IO Listen
+walletListenFromEnv envFromText = envFromText "CARDANO_WALLET_PORT" >>= \case
```

Comment:
(Adding `envFromText` as a parameter makes it possible to disentangle the packages, but it also makes `walletListenFromEnv` more difficult to use. 🤔

I would probably depend on `cardano-wallet-application-extras` for `envFromText`, once we have it.)